src/pudl/transform/eia861.py

```diff
@@ -692,63 +712,57 @@ def _clean_nerc_add_rows(df, idx_cols, idx_no_nerc):
         df (pandas.DataFrame): A DataFrame with the column 'nerc_region' to be
             cleaned.
         idx_cols (list): A list of the primary keys.
-        idx_no_nerc (list): A list of the primary keys, not including nerc_region.
 
     Returns:
         pandas.DataFrame: A DataFrame with correct and clean nerc regions.
 
     """
+    idx_no_nerc = idx_cols.copy()
+    if 'nerc_region' in idx_no_nerc:
+        idx_no_nerc.remove('nerc_region')
+
     # Split raw df into primary keys plus nerc region and other value cols
     nerc_df = df[idx_cols]
     other_df = df.drop('nerc_region', axis=1).set_index(idx_no_nerc)
 
+    # Backfill the utility with strange numeric nerc region
+    nerc_df.loc[nerc_df.utility_id_eia == 55959, 'nerc_region'] = 'MRO'
+
```

Comment:
This bit is throwing that weird loc statement error.

Reply:
Ahh. I think your problem is actually the line above ` nerc_df = df[idx_cols]`. If you change this to a `loc` selection or make a copy it should stop the warning.

Also, I think you could probably add the numeric nerc region into the `NERC_SPELLCHECK`. Not necessary, but it might this function slightly simpler.

Reply:
Good point! Thanks